src/pages/configurePatch/configurePatchCore/ConfigureTasks.tsx

```diff
@@ -7,6 +7,8 @@ import Checkbox from "@leafygreen-ui/checkbox";
 import { css } from "@emotion/core";
 import isEmpty from "lodash/isEmpty";
 import { Button } from "components/Button";
+import get from "lodash/get";
```

Comment:
I believe lodash.get is utilizing a dependency listed our package.json dependency list whereas lodash/get is utilizing a peer dependency. I can tell because if uninstall lodash.every but replace all lodash.every imports to lodash/every, the app will still compile.